Boost::System was converted to a header-only library in Boost 1.89 (in
https://github.com/boostorg/system/commit/7a495bb46d7ccd808e4be2a6589260839b0fd3a3,
more specifically), so don't try to link against it.
--
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 51ff7cf..5c04547 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,7 +29,7 @@ information. Otherwise, CTest must "scrape" the build output log for
 diagnostics.
 ]])
 
-find_package(Boost REQUIRED COMPONENTS filesystem stacktrace_basic system)
+find_package(Boost REQUIRED COMPONENTS filesystem stacktrace_basic)
 find_package(Catch2 REQUIRED)
 find_package(Cuda REQUIRED)
 find_package(fmt REQUIRED)
diff --git a/common/ChildProcess/CMakeLists.txt b/common/ChildProcess/CMakeLists.txt
index 02fc5b9..d4e7dfd 100644
--- a/common/ChildProcess/CMakeLists.txt
+++ b/common/ChildProcess/CMakeLists.txt
@@ -36,7 +36,6 @@ target_link_libraries(childprocess PUBLIC
     dcgm_common
     dcgm_logging
     fmt::fmt
-    Boost::system
     Boost::filesystem
 )
 
diff --git a/common/RemoteConn/CMakeLists.txt b/common/RemoteConn/CMakeLists.txt
index 01c0f3b..65f8806 100644
--- a/common/RemoteConn/CMakeLists.txt
+++ b/common/RemoteConn/CMakeLists.txt
@@ -24,8 +24,7 @@ target_link_libraries(remoteconn PUBLIC
     dcgm
     dcgm_logging
     fmt::fmt
-    Boost::system
     Boost::filesystem
 )
